build.ps1

```diff
@@ -404,18 +412,16 @@ function Build-CMakeProject {
     }
 
     # Debug Information
-    if ($IsRelease) {
-      $SwiftArgs.Add("-gnone") | Out-Null
+    if ($GenerateDebugInfo) {
+      if ($SwiftDebugFormat -eq "dwarf") {
+        $SwiftArgs.Add("-g -Xlinker /DEBUG:DWARF -use-ld=lld-link") | Out-Null
+      } else {
+        $SwiftArgs.Add("-g -debug-info-format=codeview -Xlinker -debug") | Out-Null
+      }
```

Comment:
One thing that was nice about the slightly more repetitive variant is that we could easily see the difference between the compile and link flags.  We do loose that here though we gain some brevity.

Reply:
I'm not super opinionated here but I did get annoyed by all those `| Out-Null`'s.